storage/backend/ceph/util.go

```diff
@@ -13,7 +14,13 @@ func (c *Driver) MountPath(do storage.DriverOptions) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return filepath.Join(c.mountpath, do.Volume.Params["pool"], volName), nil
+
+	poolName, ok := do.Volume.Params["pool"].(string)
+	if !ok {
+		return "", errored.Errorf("Expected string type for driver.pool")
```

Comment:
this message should be a constant. we should not have it in many places.

sorry I did not realize this before.